src/main/java/quiz/model/QuizRound.java

```diff
@@ -82,7 +125,13 @@ public void askQuestion(StringBuilder speechText) {
 		int asked = this.askedQuestions.length;
 		this.askedQuestions = Arrays.copyOf(this.askedQuestions, asked + 1);
 		this.askedQuestions[asked] = q;
-		speechText.append(players[asked % players.length].name + ": ");
+		if(players.length == 1) {
+			speechText.append(players[asked % players.length].name);

```

Comment:
Nein, so war das nicht gedacht. Bei nur einem Spieler soll *gar kein* Name ausgegeben werden. Auch nicht wenn es der leere String ist.

Die Bedingung kann `players.length` mit `1` vergleichen, besser aber wäre es den `name` auf `isEmpty` zu überprüfen. Die Logik, *ob und welche* Spielernamen bei welcher Zahl von Spielern ausgegeben werden findet sich dann nur in `createPlayers`. Dafür war der leere String gedacht.